easybuild/tools/github.py

```diff
@@ -835,6 +877,147 @@ def is_patch_for(patch_name, ec):
     return soft_name
 
 
+def check_pr_eligible_to_merge(pr_data):
+    """
+    Check whether PR is eligible for merging.
+
```

Comment:
We should stick to one style, and I generally prefer having a blank line there as well, but the only way to make this consistent is to have an automated style check for it...